java/INSTALL

```diff
@@ -103,16 +108,16 @@ general overview of OpenDDS.
      use paths with no spaces (true on Unix too but spaces are rarely seen).
      The top level of the OpenDDS distribution is a directory that contains
      the subdirectory named dds.
-
+     
   3. Configure and build OpenDDS 
 
      Build as in $DDS_ROOT/install, but add --java option
 
-     Unix =>
-       ./configure --java=<full path to the JDK>
-     Windows => 
-       configure --java=<full path to the JDK>
-
+     Unix => cd $DDS_ROOT
+             ./configure --java
+             make       
+     Windows => configure --java=<full path to the JDK>
```

Comment:
Search for the string "sub which {" in the configure script.  You can call this on all platforms.